frontend/packages/dev-console/src/components/formik-fields/RadioButtonField.scss

```diff
@@ -1,5 +1,16 @@
 .odc-radio-button {
-  &__helper-text {
-    padding: var(--pf-global--spacer--md) 0;
-  }
+  padding-left: var(--pf-global--spacer--sm);
+}
+
+.odc-radio-button input[type='radio'] {
+  margin-top: 0;
+}
+
+.odc-radio-button__helper-text {
+  padding: var(--pf-global--spacer--md) 0;
+  line-height: 1em;
+}
+
+.odc-radio-button__displayed-field {
+  padding: var(--pf-global--spacer--md) 0 var(--pf-global--spacer--md) 1.8rem;
 }
```

Comment:
Please update this, making use of our SCSS style. See `frontend/packages/dev-console/src/components/import/section/ResourceSection.scss` for an example of how to nest items.

If you're unfamiliar with the inner workings of SCSS and the parent selector, please see https://sass-lang.com/documentation/style-rules/parent-selector.

Reply:
Thanks for the reference. Was not sure if we can nest selectors as well.

Reply:
Ah yeah, I guess documentation only goes so far.

So the way SCSS nesting works is:
```
.my-class {
  .my-inner-class { }

  &.my-modifier { }

  input[type=radio] { }
}
```

Gives us this CSS in the end (if there were properties inside them):
```
// top-level properties for the first dom element
.my-class {}

 // two dom elements, one nested inside the other
.my-class .my-inner-class {}

 // one dom element with a modifier class
.my-class.my-modifier

 // two dom elements, inner one being an radio input
.my-class input[type=radio] {}
```

`&` is merely append to the selector (aka don't use a space)